README.md

```diff
@@ -181,27 +181,10 @@ The tests require a Postgres database to which to connect to. Here's one way to
 * Run tests within the Django container: `docker-compose exec django pytest`.
 
 
-## Issue tracking
```

Comment:
These shouldn't be removed. See [City of Helsinki documentation practises](https://helsinkisolutionoffice.atlassian.net/wiki/spaces/OS/pages/7738589197/Dokumentaatiok+yt+nn+t+documentation+practices).

Reply:
Ah yes, link to issue tracking. But the documentation practices also says that under "Links to public information", so I guess the non-public link can be removed?

Reply:
Yes, I would assume so.

Reply:
According to the guidelines publicly accessible environments should also be listed, so I think I'll resurrect (and correct) those too.